### DISCUSSION TITLE
Any suggestion for hardware requirement in this case?

### DISCUSSION THREAD

**Discussion by nvcnvn** [valhalla/valhalla#4181] [General]
I'm gonna to test with this routing scenario:

- Bidirectional A*, auto cost, car only
- Routing between locations in a single city (using the map of Vietnam https://download.geofabrik.de/asia/vietnam.html)
- 50 requests per second with random source&destination
- Around 10 km between source and destination

Any draft estimation for starting point with hardware resource in this case?

Thank you!

**Comment by kevinkreiser** [comment]
10km is nothing you can do that on a single machine and a small number of threads. do a bit of testing on your specif7c hardware to be sure though
